src/toast/ops/noise_model.py

```diff
@@ -834,7 +835,7 @@ def _exec(self, data, detectors=None, **kwargs):
                     }
                     msg = f"obs {obs.name}|{group}: flagged {len(group_flags)}"
                     msg += " outlier detectors"
-                    log.info(msg)
+                    log.debug(msg)
```

Comment:
I think I would still like some sort of tally of bad detectors at the `info` level. Perhaps not per observation. That can be added at some point later.